torch/_torch_docs.py

```diff
@@ -5390,6 +5390,11 @@ def parse_kwargs(desc):
 Returns a tuple containing the LU factorization and pivots. Pivoting is done if
 :attr:`pivot` is set.
 
+.. note::
+    LU factorization without pivoting is not available for CPU, and attempting
```

Comment:
nit: this is a bit difficult to parse because of all the negatives.  Instead of saying "without pivoting" you could say "with pivot=False."

Reply:
Sure, thank you for the suggestion.